tests/model_selection/test_incremental.py

```diff
@@ -414,10 +414,14 @@ def test_numpy_array(c, s, a, b):
     X, y = make_classification(n_samples=100, n_features=5, chunks=(10, 5))
     X, y = yield c.compute([X, y])
     model = SGDClassifier(tol=1e-3, penalty="elasticnet")
-    params = {"alpha": np.logspace(-2, 10, 10), "l1_ratio": np.linspace(0.01, 1, 20)}
+    params = {
+        "alpha": np.logspace(-5, -3, 10),
```

Comment:
Why is `alpha` changed from `logspace(-2, 10)` to `logspace(-5, -3)`?

Reply:
Because `logspace(-2, 10)` creates values from `1e-2` to `1e10`. That's unreasonable, especially because `alpha` is a weight on the regularization term. `alpha=1e10` will result in all of `model.coef_` being 0. Small `alpha`s are more realistic and are provided.